src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilStaffOrgRolesTest.java

```diff
@@ -0,0 +1,146 @@
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@RunWith(MockitoJUnitRunner.class)
+class DroolCivilStaffOrgRolesTest extends DroolBase {
+
+    @Test
+    void shouldReturnCivilCaseworkerMappings() {
+
+        judicialAccessProfiles.clear();
+        judicialOfficeHolders.clear();
+
+        CaseWorkerAccessProfile cap = UserAccessProfileBuilder.buildUserAccessProfileForRoleId2();
+        cap.setServiceCode("AAA6");
+        cap.setSuspended(false);
+        cap.setRegionId("region1");
+
+        allProfiles.add(cap);
+
+        //Execute Kie session
+        buildExecuteKieSession(getFeatureFlags("civil_wa_1_0", true));
+
+        //Extract all created role assignments using the query defined in the rules.
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        QueryResults queryResults = (QueryResults) results.getValue(ROLE_ASSIGNMENTS_RESULTS_KEY);
+        for (QueryResultsRow row : queryResults) {
+            roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
+        }
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+        assertEquals(2, roleAssignments.size());
+        assertThat(roleAssignments.stream().map(RoleAssignment::getRoleName).collect(Collectors.toList()),
+                containsInAnyOrder("tribunal-caseworker", "hmcts-legal-operations"));
+        roleAssignments.forEach(r -> {
+            assertEquals("LEGAL_OPERATIONS", r.getRoleCategory().toString());
+            assertEquals("ORGANISATION", r.getRoleType().toString());
+        });
+
+        roleAssignments.stream().filter(c -> c.getGrantType().equals(GrantType.STANDARD)).collect(Collectors.toList())
+                .forEach(r -> {
+                    assertEquals("CIVIL", r.getAttributes().get("jurisdiction").asText());
+                    assertEquals("decision_making_work", r.getAttributes().get("workTypes").asText());
+                    assertEquals("region1", r.getAttributes().get("region").asText());
+                    assertEquals(cap.getPrimaryLocationId(), r.getAttributes().get("primaryLocation").asText());
+                });
+    }
+
+    static Stream<Arguments> generateData() {
+        return Stream.of(
+                Arguments.of("3", Arrays.asList("hmcts-admin", "hearing-centre-team-leader"),
+                        2, Collections.singletonList("hearing_work,access_requests")),
+                Arguments.of("10", Arrays.asList("hmcts-admin", "ctsc", "national-business-centre"),
+                        3, Arrays.asList("routine_work", "routine_work")),
+                Arguments.of("9", Arrays.asList("hmcts-admin", "ctsc-team-leader", "nbc-team-leader"),
+                        3, Arrays.asList("routine_work,access_requests", "routine_work,access_requests")),
+                Arguments.of("4", Arrays.asList("hmcts-admin", "hearing-centre-admin"),
+                        2, Collections.singletonList("hearing_work"))
```

Comment:
Is there any other place have remaining scenarios?
I can't see test cases for roleId 2, 6,11